ui/app/mainui/AppMain.qml

```diff
@@ -836,8 +836,24 @@ Item {
 
     Settings {
         id: appMainLocalSettings
+        category: "AppMainLocalSettings_%1".arg(allContacsAdaptor.selfPubKey)
         property var whitelistedUnfurledDomains: []
         property bool introduceYourselfPopupSeen
+        property var recentEmojis
```

Comment:
just to dobule-check. Aren't recent emojis intended to be synced across multiple devices?

Reply:
They never were. It would be a nice improvement, but it's not really a priority